systemtest/src/test/java/io/strimzi/systemtest/OpenShiftTemplatesTest.java

```diff
@@ -80,37 +78,84 @@ public void testStrimziPersistent() throws IOException {
         assertEquals("persistent-claim", mapper.readTree(cmData.get("zookeeper-storage")).get("type").asText());
     }
 
+    @Test
+    public void testStrimziEphemeralWithCustomParameters() {
+        String clusterName = "test-ephemeral-with-custom-parameters";
+        oc.newApp("strimzi-ephemeral", map("CLUSTER_NAME", clusterName,
+                "ZOOKEEPER_HEALTHCHECK_DELAY", "30",
+                "ZOOKEEPER_HEALTHCHECK_TIMEOUT", "10",
+                "KAFKA_HEALTHCHECK_DELAY", "30",
+                "KAFKA_HEALTHCHECK_TIMEOUT", "10",
+                "KAFKA_DEFAULT_REPLICATION_FACTOR", "2",
+                "KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "5",
+                "KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", "5"));
+
+        ConfigMap cm = client.configMaps().inNamespace(NAMESPACE).withName(clusterName).get();
+        assertNotNull(cm);
+        Map<String, String> cmData = cm.getData();
```

Comment:
For the sake of clarity wrt my earlier comment: Here it's OK to make assertions about the cm data (rather than the Kafak broker pods which get deployed by the CC according to the CM), because this is really a test of the OpenShift *template*, rather than the CC itself.